spec/Entity/ProductBundleSlotSpec.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace spec\SolutionDrive\SyliusProductBundlesPlugin\Entity;
+
+use PhpSpec\ObjectBehavior;
+use SolutionDrive\SyliusProductBundlesPlugin\Entity\ProductBundleInterface;
+use SolutionDrive\SyliusProductBundlesPlugin\Entity\ProductBundleSlot;
+use SolutionDrive\SyliusProductBundlesPlugin\Entity\ProductBundleSlotInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Resource\Model\ResourceInterface;
+
+class ProductBundleSlotSpec extends ObjectBehavior
+{
+    function it_is_initializable()
+    {
+        $this->shouldHaveType(ProductBundleSlot::class);
+    }
+
+    function it_implements_product_bundle_slot_interface()
+    {
+        $this->shouldImplement(ProductBundleSlotInterface::class);
+    }
+
+    function it_implements_resource_interface()
+    {
+        $this->shouldImplement(ResourceInterface::class);
+    }
+
+    function it_has_a_name()
+    {
+        $this->setName('top_slot');
+        $this->getName()->shouldReturn('top_slot');
+    }
+
+    function it_has_a_position()
+    {
+        $this->setPosition(42);
+        $this->getPosition()->shouldReturn(42);
+    }
+
+    function it_can_be_assigned_to_a_bundle(ProductBundleInterface $bundle)
```

Comment:
Not sure if we need to get the bundle when handling a slot. I have the feeling that getting slots from a bundle is the more usually case, but maybe someone has an idea about a case where we need to get the bundle in the context of a slot.

Reply:
I've simply made a bidirectional association. We can drop it on this side, but I think it is not that important.